src/commands/referendum/index.ts

```diff
@@ -0,0 +1,88 @@
+import { Client, Message, User } from "discord.js";
+import _ from "lodash";
+import { Command } from "../command.interface";
+import { hasVoted, isPoll } from "./helper";
+import {ReferendumCommandKeyList} from "./referendum.interface";
+// only one poll at a time
+var voted: string[] = [];
+var votes: { [key: string]: number } = {};
+const poll: Command = { // starts vote, must supply options
+  commandDescription: "starts vote, must supply candidates to be voted on",
+  commandCallback: (
```

Comment:
I'm going to make this promise based instead cause then the testing would be easier